### PR TITLE
Fix event titles and links in doc generator

### PR DESCRIPTION
**Pull request by ryanchristo**
## Description

This pull request fixes broken links to events and changes the event titles from links to inline code.

## Other changes

Fixes minor typo in `claimColonyFunds` description.

### CODE REVIEW

File: packages/colony-js-client/scripts/docgen.js [hunk 1]
```diff
@@ -284,7 +284,7 @@ function getEventName(p) {
 }
 
 function getNestedEventDescription(event) {
-  return `Contains the data defined in [${event.name}](#events-${event.name})`;
+  return `Contains the data defined in [${event.name}](#events${event.name.toLowerCase()}addlistener${printArgs(event.args).toLowerCase().replace(/[,\{\}]/g,'-').replace(/ /g,'')}------)`;
```
Comment (JamesLefrere) [inline packages/colony-js-client/scripts/docgen.js]: What's the deal with the `------`? Also protip with regex, `\w` matches any whitespace character.

Comment (JamesLefrere) [reply packages/colony-js-client/scripts/docgen.js]: Ah I think I get it now; it matches the characters in `=>{/*...*/})`

Comment (ryanchristo) [reply packages/colony-js-client/scripts/docgen.js]: I'm not entirely sure why `------` but this is how Gatsby compiles everything after the event arguments. Manually tested with docs site running locally.

Comment (JamesLefrere) [reply packages/colony-js-client/scripts/docgen.js]: Ok, let's go with it 👍

**Comment by JamesLefrere** [comment]
Beautiful!

**Pull request merged**